src/pages/shop/elements/ShopItems.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import { SHOP_ITEMS } from '../../../constants';
+
+export const ShopItems = ({ setAmountOfItems, currentSection }) => {
+  const cartList = JSON.parse(localStorage.getItem('cartList')) === null ? [] : JSON.parse(localStorage.getItem('cartList'));
```

Comment:
Why you equal with "null". Can you do smth like this 
localStorage.getItem('cartList')) ? JSON.parse(localStorage.getItem('cartList') : []